packages/evidence/cli.js

```diff
@@ -349,12 +363,18 @@ prog
 prog
 	.command('preview')
 	.describe('preview the production build')
+	.option('--debug', 'Enables verbose console logs')
```

Comment:
Note: pretty sure the omission of the `--debug` option here is a bug, it's referenced in the action